router/image/templates/nginx.conf

```diff
@@ -39,6 +39,10 @@ http {
     gzip_vary {{ or (getv "/deis/router/gzipVary") "on" }};
     {{ end }}
 
+    # DNS servers to query for address resolution. Needed to resolve OCSP responder.
+    resolver 8.8.8.8 8.8.4.4 valid=300s;
```

Comment:
I'm concerned with hard-coding this.. I could imagine scenarios where users want to use an internal DNS server or use clusters that are completely cut-off from the internet, and this would break.